DEVELOP.md

```diff
@@ -1,9 +1,14 @@
 # Contributing to the RAPIDS devcontainers
 
+## Features
+
 From the official devcontainer [documentation on Features](https://containers.dev/implementors/features/):
 > Development container "Features" are self-contained, shareable units of installation code and development container configuration.
 
-In short, a "feature" is a layer in a Dockerfile which encapsulates a reusable bit of logic executed when building a Docker image.
+In short, a "feature" is a layer in a Dockerfile which encapsulates a reusable
+bit of logic executed when building a Docker image. It is not a docker layer to
+put on top of or copied into other layers. It is the script that creates a
+layer.
```

Comment:
I don't like this edit. There is no such thing as "a docker layer to put on top of or copied into other layers." The feature is _literally_ a layer in an image. If you think it's important to distinguish that a feature is not an image, we should say that instead.

Reply:
Indeed, thanks. The wording on this stuff needs to be very precise.

Reply:
Does this read better?

> In short, a "feature" is a reusable bit of logic executed when building a Docker image. Each "feature" specified becomes a `RUN` statement in a temporary Dockerfile, and as such each "feature" results in an image layer.

Reply:
Yeah, that sounds great!